docs/source/comparison.rst

```diff
@@ -0,0 +1,69 @@
+Comparison with Other Frameworks
+================================
+
+A table for quick comparison
+----------------------------
+
+This table compares Chainer and other popular deep learning frameworks.
```

Comment:
It seems compare do not have such an usage. "... compare Chainer with other popular deep learning frameworks"
http://www.oxforddictionaries.com/definition/english/compare